BridgeAppSDK/SBABridgeInfo.swift

```diff
@@ -93,27 +92,39 @@ public class SBABridgeInfoPList : NSObject, SBABridgeInfo {
         self.plist = plist
     }
     
-    public var appStoreLinkURL: NSURL! {
-        guard let appStoreLinkURLString = self.plist["appStoreLinkURL"] as? String,
-            let url = NSURL(string: appStoreLinkURLString)
-        else {
-            return NSBundle.mainBundle().appStoreLinkURL()
-        }
-        return url
+    public var appStoreLinkURLString: String? {
+        return  self.plist["appStoreLinkURL"] as? String
     }
     
-    public var emailFormatForRegistrationViaExternalId: String? {
-        return self.plist["emailFormatForRegistrationViaExternalId"] as? String
+    public var emailForLoginViaExternalId: String? {
+        return self.plist["emailForLoginViaExternalId"] as? String
     }
     
-    public var passwordFormatForRegistrationViaExternalId: String? {
-        return self.plist["passwordFormatForRegistrationViaExternalId"] as? String
+    public var passwordFormatForLoginViaExternalId: String? {
+        return self.plist["passwordFormatForLoginViaExternalId"] as? String
     }
     
     public var testUserDataGroup: String? {
         return self.plist["testUserDataGroup"] as? String
     }
+}
 
+extension SBABridgeInfo {
+    
+    public var emailFormatForLoginViaExternalId: String? {
+        guard let email = self.emailForLoginViaExternalId, let range = email.rangeOfString("@") else {
+            return nil
+        }
+        return email.stringByReplacingCharactersInRange(range, withString: "+%@@")
+    }
     
+    public var appStoreLinkURL: NSURL! {
+        guard let appStoreLinkURLString = self.appStoreLinkURLString,
+            let url = NSURL(string: appStoreLinkURLString)
+            else {
+                return NSBundle.mainBundle().appStoreLinkURL()
+        }
```

Comment:
It's definitely one of the things I am struggling with in swift especially since the lines can get so long.